res/themes/light/css/_fonts.pcss

```diff
@@ -1,149 +1,6 @@
 /* the 'src' links are relative to the bundle.css, which is in a subdirectory.
  */
 
-/* Inter unexpectedly contains various codepoints which collide with emoji, even
-   when variation-16 is applied to request the emoji variant.  From eyeballing
-   the emoji picker, these are: 20e3, 23cf, 24c2, 25a0-25c1, 2665, 2764, 2b06, 2b1c.
-   Therefore we define a unicode-range to load which excludes the glyphs
-   (to avoid having to maintain a fork of Inter). */
```

Comment:
Is this not necessary anymore?

Reply:
Indeed, not possible but also not necessary. fontsource adds the correct unicode ranges for each font-face definition

Reply:
Bonus!